Resources/mgmt-plane/L3N1YnNjcmlwdGlvbnMve30vcmVzb3VyY2Vncm91cHMve30vcHJvdmlkZXJzL21pY3Jvc29mdC5uZXR3b3JrL2V4cHJlc3Nyb3V0ZXBvcnRzL3t9/2022-01-01.xml

```diff
@@ -70,7 +70,7 @@
                       </enum>
                     </prop>
                     <prop readOnly="True" type="string" name="etherType"/>
-                    <prop type="array<object>" name="links" identifiers="name">
```

Comment:
the identifiers removed

Reply:
Please fix the swagger and regenerate the model

Reply:
Will do that.